src/PrestaShopBundle/Resources/config/services/bundle/services.yml

```diff
@@ -96,13 +96,20 @@ services:
     public: false
     tags: [ 'container.env_var_processor' ]
 
-  PrestaShopBundle\Api\StateProvider\HookStatusProvider:
+  PrestaShopBundle\Api\Provider\QueryProvider:
```

Comment:
All the providers/services associated with ApiPlatform should also me moved in the appropriate namespace